example_16/bringup/launch/demo_test_helper.py

```diff
@@ -22,25 +22,8 @@
 class DiffbotChainedControllersTest(Node):
     def __init__(self):
         super().__init__("diffbot_chained_controllers_demo_helper_node")
-        # Enable feedforward control via service call
-        self.client_left_ = self.create_client(
-            SetBool, "/pid_controller_left_wheel_joint/set_feedforward_control"
-        )
-        self.client_right_ = self.create_client(
-            SetBool, "/pid_controller_right_wheel_joint/set_feedforward_control"
-        )
         self.publisher_ = self.create_publisher(TwistStamped, "/cmd_vel", 10)
 
-    def set_feedforward_control(self):
-        while not self.client_left_.wait_for_service(timeout_sec=1.0):
-            self.get_logger().info(
-                "Waiting for left feedforward control service to be available..."
-            )
-        while not self.client_right_.wait_for_service(timeout_sec=1.0):
-            self.get_logger().info(
-                "Waiting for right feedforward control service to be available..."
-            )
-
         request_left = SetBool.Request()
```

Comment:
Please remove the whole set_feedforward_control method, up to line 55. This is not needed any more?